### ISSUE TITLE
`diffcov` is failing on fork PRs

### ISSUE THREAD

**Issue by FFY00** [python/importlib_resources#198]
Looks like `diffcov` is failing. I am pretty sure this is due to the repo being a fork.

```
diffcov create: /home/runner/work/importlib_resources/importlib_resources/.tox/diffcov
diffcov installdeps: coverage>=4.5, diff_cover
diffcov develop-inst: /home/runner/work/importlib_resources/importlib_resources
diffcov installed: coverage==5.3,diff-cover==4.0.1,-e git+python/importlib_resources@9a784bc22d6d2a36694c3dd8dd5b4206f4e6fe4b#egg=importlib_resources,inflect==4.1.0,Jinja2==2.11.2,jinja2-pluralize==0.3.0,MarkupSafe==1.1.1,pluggy==0.13.1,Pygments==2.7.1
diffcov run-test-pre: PYTHONHASHSEED='3388631335'
diffcov run-test: commands[0] | python -m unittest discover
......................................................................................................s........................
----------------------------------------------------------------------
Ran 127 tests in 0.060s

OK (skipped=1)
diffcov run-test: commands[1] | python -m coverage run --rcfile=/home/runner/work/importlib_resources/importlib_resources/coverage.ini -m unittest discover
......................................................................................................s........................
----------------------------------------------------------------------
Ran 127 tests in 0.044s

OK (skipped=1)
diffcov run-test: commands[2] | python -m coverage combine --rcfile=/home/runner/work/importlib_resources/importlib_resources/coverage.ini
diffcov run-test: commands[3] | python -m coverage xml --rcfile=/home/runner/work/importlib_resources/importlib_resources/coverage.ini
diffcov run-test: commands[4] | diff-cover coverage.xml --html-report diffcov.html
Traceback (most recent call last):
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/bin/diff-cover", line 8, in <module>
    sys.exit(main())
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/diff_cover_tool.py", line 216, in main
    percent_covered = generate_coverage_report(
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/diff_cover_tool.py", line 184, in generate_coverage_report
    reporter.generate_report(output_file)
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/report_generator.py", line 251, in generate_report
    report = template.render(self._context())
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/report_generator.py", line 300, in _context
    context = super().report_dict()
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/report_generator.py", line 183, in report_dict
    src_stats = {src: self._src_path_stats(src) for src in self.src_paths()}
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/report_generator.py", line 85, in src_paths
    for src, summary in self._diff_violations().items()
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/report_generator.py", line 178, in _diff_violations
    for src_path in self._diff.src_paths_changed()
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/diff_reporter.py", line 151, in src_paths_changed
    diff_dict = self._git_diff()
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/diff_reporter.py", line 200, in _git_diff
    for diff_str in self._get_included_diff_results():
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/diff_reporter.py", line 173, in _get_included_diff_results
    included = [self._git_diff_tool.diff_committed(self._compare_branch)]
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/git_diff.py", line 66, in diff_committed
    return execute(self._default_git_args + self._default_diff_args + [diff_range])[
  File "/home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/lib/python3.9/site-packages/diff_cover/command_runner.py", line 45, in execute
    raise CommandError(stderr)
diff_cover.command_runner.CommandError: fatal: ambiguous argument 'origin/master...HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

ERROR: InvocationError for command /home/runner/work/importlib_resources/importlib_resources/.tox/diffcov/bin/diff-cover coverage.xml --html-report diffcov.html (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   diffcov: commands failed
```
https://github.com/python/importlib_resources/pull/197/checks?check_run_id=1298448406

**Comment by FFY00** [comment]
We probably want to do the same we did in the CPython CI.

https://github.com/python/cpython/blob/780151479f6f417838fc9e797864cf0b807604a9/.github/workflows/build.yml#L48

**Comment by FFY00** [comment]
Well, that comes from `diff_cov`. Fixed in #199, it's not the most efficient fix but is fine and we have no other alternative since we use `diff_cov`.
